### PR TITLE
Dashboard editing

### PR DESCRIPTION
**Pull request by Abbey-Wilkinson**
- Added pymssql to requirements.
- Changed docstring.

### CODE REVIEW

File: dashboard/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+RUN pip3 install -r requirements.txt
+
+EXPOSE 8501
+
+COPY database.py .
+COPY utilities.py .
+COPY visualisations.py .
+COPY app.py .
+
+CMD streamlit run app.py
```
Comment (caithomas6789) [inline dashboard/Dockerfile:15]: Dockerfile looks good.

File: dashboard/app.py [hunk 2]
```diff
@@ -63,10 +60,16 @@ def get_main_body(plants) -> None:
         st.altair_chart(get_latest_temperature_readings(
             plants[name_in_selected_plants], sort_ascending_temp), use_container_width=True)
 
+        st.altair_chart(get_temperature_over_time(
+            plants[name_in_selected_plants]), use_container_width=True)
+
```
Comment (OscarB28) [inline dashboard/app.py:65]: definitely useful to see this

File: dashboard/app.py [hunk 3]
```diff
@@ -63,10 +60,16 @@ def get_main_body(plants) -> None:
...
     with body_cols[1]:
         st.altair_chart(get_latest_soil_moisture_readings(
             plants[name_in_selected_plants], sort_ascending_moisture), use_container_width=True)
 
+        st.altair_chart(get_soil_moisture_over_time(
+            plants[name_in_selected_plants]), use_container_width=True)
```
Comment (caithomas6789) [inline dashboard/app.py:71]: A chart to show soil moisture over time is a great idea.

File: dashboard/visualisations.py [hunk 4]
```diff
@@ -16,17 +18,16 @@ def get_latest_temperature_readings(plants: DataFrame, sort_ascending):
     latest_data = get_latest_data(plants)
 
     latest_data["Plant Name"] = latest_data[["plant_name"]]
-    latest_data["Temperature"] = latest_data[["temperature"]]
+    latest_data["Temperature (°C)"] = latest_data[["temperature"]]
```
Comment (caithomas6789) [inline dashboard/visualisations.py:21]: Including temp unit makes the graph a lot clearer.

File: dashboard/visualisations.py [hunk 5]
```diff
@@ -67,7 +68,20 @@ def get_temperature_over_time(plants: DataFrame):
...
+        "temperature"].mean().reset_index()
+
+    line_chart = alt.Chart(temp_over_time).mark_line().encode(
+        x=alt.X('Time', title='Time (hr)').scale(zero=False),
+        y=alt.Y('temperature:Q', title='Temperature °C').scale(zero=False),
+        color=alt.Color('Plant Name:N', legend=None).scale(scheme='greens')
+    ).properties(
+        title='Temperature of Plants over time'
+    )
+    return line_chart
```
Comment (caithomas6789) [inline dashboard/visualisations.py:84]: This graph looks good.

**Review by caithomas6789 — APPROVED** [review]

File: dashboard/app.py [hunk 6]
```diff
@@ -63,10 +60,16 @@ def get_main_body(plants) -> None:
...
+
```
Comment (OscarB28) [inline dashboard/app.py:72]: This is useful too

File: dashboard/visualisations.py (hunk 4 above)
Comment (OscarB28) [inline dashboard/visualisations.py:21]: scale is important

**Review by OscarB28 — APPROVED** [review]
Looks amazing!

**Pull request merged**